src/components/StoryAccordion.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+
+interface AccordionItem {
+  label: string;
+  content: string;
+}
+
+interface StoryAccordionProps {
+  items: AccordionItem[];
+}
+
+const StoryAccordion: React.FC<StoryAccordionProps> = ({ items }) => {
+  const [accordionItems, setAccordionItems] = useState(
+    items.map((item) => ({ ...item, isOpen: false }))
```

Comment:
I suggest having isOpen as a separate array

```js
const [isOpens, setIsOpens] = useState([])

-item.isOpen
+isOpens[index]
```
toggle would be just
```js

  const toggleAccordion = (index: number) => {
    setIsOpens(arr => arr.map((item, idx) => idx === index ? !item : item))
  }
```

I am also glad that we are using array, not a single value for currently visible tab as it's done [here](https://github.com/salesforce/design-system-react/blob/v0.10.58/components/accordion/index.jsx)

if possible please extract isOpens as array, if not clear - we can merge

Reply:
As you suggested, I refactored the accordion state management.
Improvements:
- refactored state management to use an isOpens array for tracking open states.
- updated toggleAccordion function for better readability.
- corrected usage of template literals for accurate rendering.